pkg/types/workspace.go

```diff
@@ -50,3 +50,7 @@ type WorkspaceInfo struct {
 	Projects         []*ProjectInfo `json:"projects"`
 	ProviderMetadata string         `json:"providerMetadata,omitempty"`
 } // @name WorkspaceInfo
+
+type WorkspaceListOptions struct {
```

Comment:
This should not belong in the `types` package. Types are something core to Daytona and `WorkspaceListOptions` is something only related to the API controller.

Reply:
removed, since it is no longer required.